CondCore/AlignmentPlugins/test/testTrackerAlignmentPayloadInspector.cpp

```diff
@@ -22,8 +22,8 @@ int main(int argc, char** argv) {
   std::string connectionString("frontier://FrontierProd/CMS_CONDITIONS");
 
   std::string tag = "TrackerAlignment_v21_offline";
-  cond::Time_t start = boost::lexical_cast<unsigned long long>(294034);
-  cond::Time_t end = boost::lexical_cast<unsigned long long>(305898);
+  cond::Time_t start = static_cast<unsigned long long>(294034);
```

Comment:
I'm just noticing that would you have implemented something similar as in the other PR of yours,  #34976, you would have used here
```suggestion
  cond::Time_t start = std::stoull(294034);
```
etc.

What's the reason for the different approach?

Reply:
They are not string. For string conversions I have used stoi/stod/stoull for int to long/ul I used static cast since stoull will give error type not string